crawl-ref/source/directn.cc

```diff
@@ -1533,17 +1533,17 @@ static void _push_back_if_nonempty(const string& str, vector<string>* vec)
         vec->push_back(str);
 }
 
-void direction_chooser::print_target_monster_description(bool &did_cloud) const
```

Comment:
No other good way to get the targeter description because it's hidden inside a `print_...` private function.  So I pulled the description creation code out into a public `const` function and then call it from the `print_...` function.